Makefile

```diff
@@ -19,23 +20,25 @@ image:
 push:
 	docker push $(DOCKER_IMAGE_REPO):$(DOCKER_IMAGE_TAG)
 
-build:
-	go vet ./ably
-	go build -o ably-boomer ./ably
+./bin/ably-boomer:
+	go vet ./cmd/ably-boomer
+	go build -o ./bin/ably-boomer ./cmd/ably-boomer
+
+build: ./bin/ably-boomer
 
 cover: lint
 	mkdir -p ./coverage
-	go test -covermode=atomic -coverprofile=coverage/coverage.out ./ably/...
+	go test -covermode=atomic -coverprofile=coverage/coverage.out ./...
 	go tool cover -html=./coverage/coverage.out -o=./coverage/coverage.html
 
 GOLINT = $(BIN)/golint
 lint: | $(GOLINT)
-	$(GOLINT) -set_exit_status ./ably/...
+	$(GOLINT) -set_exit_status ./...
 
-test: fmt lint
-	go test ./ably/...
+test: fmt
```

Comment:
why remove the lint step from testing?

Reply:
I did remove that because I feel that they are two different things. At some point I only wanted to see it the tests sill passed after a change and I could not test because I was missing a comment on an exported function. It's also closer to the behaviour we already have in go-services.